src/api/audits.coffee

```diff
@@ -0,0 +1,129 @@
+Audit = require('../model/audits').Audit
+authorisation = require './authorisation'
+Q = require 'q'
+logger = require 'winston'
+utils = require "../utils"
+
+
+# function to construct projection object
+getProjectionObject = (filterRepresentation) ->
+  switch filterRepresentation
+    when "simpledetails"
+      # view minimum required data for audit details view
+      return {}
+    when "full"
+      # view all audit data
+      return {}
+    else
+      # no filterRepresentation supplied - simple view
+      # view minimum required data for audits
+      return { "participantObjectIdentification": 0, "activeParticipant": 0, "rawMessage": 0 }
```

Comment:
Yip by default I just want to return the 'eventIdentification' details for the listings. I think that's all that would be needed to list the audits